frontend/src/__snapshots__/App.test.js.snap

```diff
@@ -58,43 +58,45 @@ exports[`App should render correctly 1`] = `
   <div
     className="makeStyles-centerHorizontal-1"
   >
-    <div
-      className="MuiPaper-root MuiCard-root makeStyles-content-2 MuiPaper-elevation1 MuiPaper-rounded"
-    >
+    <div>
       <div
-        className="MuiCardContent-root"
+        className="MuiPaper-root MuiCard-root makeStyles-content-6 MuiPaper-elevation1 MuiPaper-rounded"
       >
-        <h3>
-          Meet fellow Googlers 
-          <em>
-            now
-          </em>
-          !
-        </h3>
-        <p>
-          Miss bumping into new faces at the office? Want an easy, fun, spontaneous way of meeting Googlers virtually? Now you can!
-        </p>
-        <p>
-          Ad-lib matches you with a fellow Googler in the queue, notifies you through email when you’ve been matched, and adds an event to your Calendar with a Meet link for you to join immediately! It also provides a starter question to get the conversation flowing!
-        </p>
+        <div
+          className="MuiCardContent-root"
+        >
+          <h3>
+            Meet fellow Googlers 
+            <em>
+              now
+            </em>
+            !
+          </h3>
+          <p>
+            Miss bumping into new faces at the office? Want an easy, fun, spontaneous way of meeting Googlers virtually? Now you can!
+          </p>
+          <p>
+            Ad-lib matches you with a fellow Googler in the queue, notifies you through email when you’ve been matched, and adds an event to your Calendar with a Meet link for you to join immediately! It also provides a starter question to get the conversation flowing!
```

Comment:
Are we still doing the starter question?

Reply:
Hmm, good point, I am not if we will have to implement this. @TcgVanguardTroll  what do you think about this? If not, I can take it out of the description.

Reply:
As per our discussion during standup, deciding to keep this in for now and will add in functionality for it in a follow up PR.